velox/exec/tests/OutputBufferManagerTest.cpp

```diff
@@ -971,6 +977,60 @@ TEST_P(AllOutputBufferManagerTest, outputBufferUtilization) {
   verifyOutputBuffer(task, OutputBufferStatus::kFinished);
 }
 
+TEST_P(AllOutputBufferManagerTest, outputBufferStats) {
+  const vector_size_t size = 100;
+  const std::string taskId = std::to_string(rand());
+  initializeTask(taskId, rowType_, kind_, 1, 1);
+  OutputBufferStats stats0 = getStats(taskId);
```

Comment:
variable names a1, a2, a2 are an anti-pattern; Here, stats0 is not needed at all as you can simply do

ASSERT_EQ(getStats(taskId).kind, kind_);

Reply:
refactored. I kind of `learned` the bad habit of a1, a2, aXX from other projects, will change it. Thanks.